docs/api/browser-window.md

```diff
@@ -558,12 +564,12 @@ Note that this is only emitted when the window is being resized manually. Resizi
 
 Emitted when the window is being moved to a new position.
 
-__Note__: On macOS this event is an alias of `moved`.
-
-#### Event: 'moved' _macOS_
+#### Event: 'moved' _macOS_ _Windows_
 
 Emitted once when the window is moved to a new position.
 
+__Note__: On macOS this event is an alias of `move`.
```

Comment:
I moved the Note into 'moved' since now the behavior only differs on this event. On Windows, 'moved' will only emit after the user has stopped moving the window. On macOS, it will always emit along with 'move'.

Reply:
After moving the note, it reads to me like this: the `moved` event is not really implemented on macOS, it is just an alias of `move` event. But I think the actual behavior is reverse?

The original note was confusing too, so I think maybe we should clearly describe the behavior in the note.

Reply:
That's how I see it as well. You had left a comment mentioning needing to figure out a proper way to dispatch move.
https://github.com/electron/electron/blob/d3f32c7502412ec143e25da41589ab923cf0b163/shell/browser/ui/cocoa/electron_ns_window_delegate.mm#L171-L177

For me, I assume `move` should follow similar behavior to how [`mousemove`](https://developer.mozilla.org/en-US/docs/Web/API/Element/mousemove_event) works. It should be emitted at each change of the position of the window.

`moved` should be emitted after moving is finished and should follow after the last `move` event. However, macOS doesn't have this behavior and will instead always emit after `move`.

Does this make sense?

Reply:
Yeah you are right.

I thought `windowDidMove` would emit when moving is finished, but after testing I found it actually keeps emitting during the moving. So moving the note does make the doc more clear 👍 .